src/components/paneler/innhold/personaliaV2/telefon.tsx

```diff
@@ -25,9 +25,15 @@ function TelefonNrMedKilde(props: { telefon: PersonaliaTelefon }) {
 
 	return (
 		<div className="overinformasjon underinformasjon">
-			<Normaltekst className="innrykk">
+			<Normaltekst className="innrykk display-flex">
 				{formaterTelefonnummer(landkode, telefonnummer)}
-				<Kopiknapp kopitekst={props.telefon.telefonNr} type="telefonnummer" />
+				<CopyToClipboard
+					copyText={props.telefon.telefonNr}
+					popoverText="Kopiert"
```

Comment:
Samme begrunnelse som i forrige kommentar:

```suggestion
					popoverText="Telefonnummer ble kopiert"
```